db/db.go

```diff
@@ -12,24 +12,16 @@ import (
 var DB *gorm.DB
 
 func init() {
+	var err error
 	name := os.Getenv("MYSQL_DATABASE")
 	host := os.Getenv("DB_HOST")
 	user := os.Getenv("DB_USER")
 	pass := os.Getenv("DB_PASS")
```

Comment:
Addressing the environment variables to a new variable, creates a memory pointer.
You can use them directly into the fmt.Sprintf

Reply:
@ike-cunha So you're suggesting something like this?
```go
dsn := fmt.Sprintf(
  "%s:%s@tcp(%s)/%s", 
  os.Getenv("DB_USER"), 
  os.Getenv("DB_PASS"), 
  os.Getenv("DB_HOST"), 
  os.Getenv("MYSQL_DATABASE")
)
```
I think that takes some readability away, but is definitely doable

Reply:
@caiobsilva you can use multiline strings. https://stackoverflow.com/a/7933487 😃

Reply:
but... look like good for me ;) using dns example. Change the code that I approve.

Reply:
done! thanks:)

Reply:
ty